hiutil.c

```diff
@@ -36,12 +36,12 @@
 #include <stdlib.h>
 #include <string.h>
 #include <sys/types.h>
+#include <sys/time.h>
 
-#ifndef WIN32
+#ifndef _WIN32
```

Comment:
I dont know windows defines, but we have checks for `WIN32` in `hiutil.h` as well.
Should we replace those also?

Reply:
Yep, I'll update 'em to consistently use _WIN32 which I believe is the correct macro everywhere.